var/spack/repos/builtin/packages/ectrans/package.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Ectrans(CMakePackage):
+    """Ectrans is the global spherical Harmonics transforms library,
+    extracted from the IFS. It is using a hybrid of MPI and OpenMP
+    parallelisation strategies. The package contains both single- and double precision
+    Fortran libraries (trans_sp, trans_dp), as well as a
+    C interface to the double-precision version (transi_dp)."""
+
+    homepage = "https://github.com/ecmwf-ifs/ectrans"
+    git = "https://github.com/ecmwf-ifs/ectrans.git"
+    url = "https://github.com/ecmwf-ifs/ectrans/archive/1.1.0.tar.gz"
+
+    maintainers("climbfuji")
+
+    version("develop", branch="develop", no_cache=True)
+    version("main", branch="main", no_cache=True)
+    version("1.2.0", sha256="2ee6dccc8bbfcc23faada1d957d141f24e41bb077c1821a7bc2b812148dd336c")
+    version("1.1.0", sha256="3c9848bb65033fbe6d791084ee347b3adf71d5dfe6d3c11385000017b6469a3e")
+
+    variant(
+        "build_type",
+        default="RelWithDebInfo",
+        description="CMake build type",
+        values=("Debug", "Release", "RelWithDebInfo"),
+    )
+
+    variant("mpi", default=True, description="Use MPI?")
+    variant("openmp", default=True, description="Use OpenMP?")
+
+    variant("double_precision", default=True, description="Support for double precision?")
+    variant("single_precision", default=True, description="Support for single precision?")
+
+    variant("mkl", default=False, description="Use MKL?")
+    variant("fftw", default=True, description="Use FFTW?")
```

Comment:
Can we have descriptions which are not questions?

Reply:
Yes, done.